src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/LaunchProfiles/ProjectLaunchProfileExtensionValueProvider.cs

```diff
@@ -71,6 +74,10 @@ public void OnSetPropertyValue(string propertyName, string propertyValue, IWrita
                     TrySetOtherProperty(launchProfile, LaunchProfileExtensions.RemoteAuthenticationModeProperty, propertyValue, string.Empty);
                     break;
 
+                case HotReloadEnabledPropertyName:
+                    TrySetOtherProperty(launchProfile, LaunchProfileExtensions.HotReloadEnabledProperty, bool.Parse(propertyValue), true);
```

Comment:
How are invalid values handled for these properties? Does the exception from bool.Parse get translated into something else in the UI?

Reply:
I'm not entirely sure what will happen. Throwing an exception here is probably not the best idea. Probably it should just ignore the input if it can't be converted to a bool, just like the `OnGet*` method ignores the stored value if it can't be converted to a bool.